testData/solutions/win32.json5

```diff
@@ -6743,6 +6840,7 @@
                         "schema": {
                             "title": "Video audio description",
                             "description": "Hear descriptions of what's happening in videos",
+                            // GPII-4207 Test to see if this `default` is getting cast correctly
```

Comment:
You are right to flag all of these with TODOs.  Previously we have used an imprecise mix of truthy and falsy stuff to represent values like these, there was a lot of it previously in the JAWS settings as well.  Until we come up with our own means of standardising these (for example, intra-application settings transforms), we should put the literal value the settings handler needs to read and write the correct value.  We can get away with using booleans, which do cast to strings, but really need to be careful about representing ones and zeros in schemas as booleans.